server.js

```diff
@@ -62,7 +62,7 @@ startServer();
 
 // TASK 3 : Use mongoose instead of mongodb driver 
```

Comment:
The code is missing an import statement for the 'admin_middleware' function, which is required in line 6.